feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -364,10 +364,9 @@ func TestUnsetElectionid(t *testing.T) {
 			if resp != test.wantStatus {
 				t.Fatalf("Incorrect status code received: want %d, got %d", test.wantStatus, resp)
 			}
-			if err != nil {
-				t.Errorf("Errors seen in ClientArbitration response: %v", err)
+			if err != nil && !deviations.P4RTSkipStreamError(ondatra.DUT(t, "dut")) {
```

Comment:
Use `dut` object directly `if err != nil && !deviations.P4RTSkipStreamError(dut)`.

Reply:
Thanks Ankur. I modified it.